src/dataIntegration/transcriptomics/preprocessing/findUsedGenesLevels.m

```diff
@@ -1,9 +1,11 @@
-function [gene_id, gene_expr] = findUsedGenesLevels(model, exprData, printLevel)
+function [gene_id, gene_expr, gene_sig] = findUsedGenesLevels(model, exprData, exprSig, printLevel)
```

Comment:
`exprSig` should go to the end of the list (again, backwards compatability).
Alternatively (and I think that's the better option):
Include `exprSig` as an optional field in `exprData` as an optional field. test whether the field is present and if, use the information from the field. if not, just don't use it.
That would make the function:
a) backward compatible
b) more flexible

Reply:
in the latest commit, significance value is an optional input in the struct `exprData`. there is also a check to see if this field exists, if yes this input is used. hope this retains the previous usage of these functions